JumpscaleCore/data/bcdb/BCDB.py

```diff
@@ -288,33 +297,33 @@ def import_(self, path, interactive=True):
                 if item.endswith("schema_hist.toml"):
                     continue
                 print(f"processing item: {item}")
+                encr_ext = j.sal.fs.getFileExtension(item.rstrip(".encr"))
```

Comment:
This block of logic can be refactored and some redundancies can be removed. I communicated them to @waleedhammam